camunda-forim-io-vue/src/components/TaskList.vue

```diff
@@ -773,7 +773,11 @@ getTaskProcessDiagramDetails(task: any) {
     })
 
     CamundaRest.getUsers(this.token, this.bpmApiUrl).then((response) => {
-      this.autoUserList = response.data.map((e: { id: number }) => (e.id));
+      this.autoUserList = []
+      response.data.forEach((element: any) => {
+         this.autoUserList.push({ "code" : element.id, "label" : element.email })
```

Comment:
This looks like the culprit which makes the variable userSelected on selecting value :

![image](https://user-images.githubusercontent.com/70306694/118016446-0582c800-b373-11eb-9ca2-a49104a4fc85.png)